src/node/install.sh

```diff
@@ -147,46 +147,17 @@ check_packages() {
         rhel)
             if ! rpm -q "$@" > /dev/null 2>&1; then
                 pkg_mgr_update
+                if [ "${ID}" = "centos" ]; then
```

Comment:
I think we can remove these changes as the tests are fixed upstream?

Reply:
The tests are currently failing.

Reply:
sure